### PR TITLE
Added Erik Serrano bio under _members folder

### PR DESCRIPTION
**Pull request by axiomcura**
Hey @gwaygenomics I added my bio into the `_members` folder. Take a look

### CODE REVIEW

File: _members/erik-serrano.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Erik Serrano
+image:
```
Comment (gwaygenomics) [inline _members/erik-serrano.md:3]: Please add image!

File: _members/erik-serrano.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+role: phd
+aliases:
+  - Erik Serrano
+  - Erik Alfonso Serrano
+  - Erik A Serrano
+  - Erik A. Serrano
+  - axiomcura
+links:
+  orcid: 0000-0002-0503-9348
```
Comment (gwaygenomics) [inline _members/erik-serrano.md:12]: Please add your [ORCID](https://orcid.org/)

File: _members/erik-serrano.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  home-page: waysciencelab.com
```
Comment (gwaygenomics) [inline _members/erik-serrano.md:13]: maybe make it your github profile page?

File: _members/erik-serrano.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  email: erik.serrano@cuanschutz.com
+  github: axiomcura
+  twitter: axiomcura
+---
+
+
+[ my  bio]
```
Comment (gwaygenomics) [inline _members/erik-serrano.md:20]: Please write a brief bio - see the site for examples

**Review by gwaygenomics — CHANGES_REQUESTED** [review]
Take a look when you get a chance

**Comment by axiomcura** [comment]
@gwaybio  added changes to my bio.

File: _members/erik-serrano.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  email: erik.serrano@cuanschutz.com
+  github: axiomcura
+  twitter: axiomcura
+---
+
+Erik Serrano is a Computational Bioscience (CPBS) PhD student at the Univeristy of Colorado, Anschutz Medical Campus (CU-AMC).
+He has joined the Way Lab in July 2022, where focuses on developing software applications to analyze cell morphology reads outs.
+Before officially joining the Way lab, Erik did conducted his last rotation where he has kick started an application know as [Cytopipe](https://github.com/WayScience/CytoPipe), which possess reproducible analytical pipeline that preprocess high-dimensional cell morphology readouts.
+
+Erik obtained his bachelors degree in Molecular Biology with a minor in chemistry at the [California State University, Northirdge (CSUN)](https://www.csun.edu/).
```
Comment (gwaybio) [inline _members/erik-serrano.md:23]:
```suggestion
Erik obtained his bachelors degree in Molecular Biology with a minor in chemistry at the [California State University, Northridge (CSUN)](https://www.csun.edu/).
```

correct?

File: _members/erik-serrano.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+Throughout his undergraduate degree, his research experience mainly pertained to developing computational methods to elucidate biological insights through the usage of protein dynamic data produced by molecular dynamic simulations.
```
Comment (gwaybio) [inline _members/erik-serrano.md:24]:
```suggestion
Throughout his undergraduate degree, he developed computational methods to elucidate biological insights from protein dynamic data produced by molecular dynamic simulations.
```

To correct grammar and brevity

File: _members/erik-serrano.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+After graduating from CSUN, Erik did his post baccalaureate at [Cedar-Sinai Medical Center](https://www.cedars-sinai.edu/research/departments-institutes/biomedical-sciences.html) where his main job was designing viral based nano-particles through the usage of computational protein modeling.
```
Comment (gwaybio) [inline _members/erik-serrano.md:25]:
```suggestion
After graduating from CSUN, Erik did his post baccalaureate at [Cedar-Sinai Medical Center](https://www.cedars-sinai.edu/research/departments-institutes/biomedical-sciences.html) where he designed viral based nano-particles through the usage of computational protein modeling.
```

brevity

File: _members/erik-serrano.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+In addition, he also developed computational methods using gene expression data to identify potential alternative entry points where the nano-particles can be introduced.
+
+In his free time, Erik likes to learn different romance languages and its origins.
+He is fluent in Spanish and is currently learning Brazilian Portuguese to become his third language.
```
Comment (gwaybio) [inline _members/erik-serrano.md:29]:
```suggestion
He is fluent in Spanish and is currently learning Brazilian Portuguese, which will become his third language.
```

Comment (gwaybio) [reply _members/erik-serrano.md:29]: You're also fluent in python! 😂

**Review by gwaybio — CHANGES_REQUESTED** [review]
Looks great! I corrected a couple typos and made grammar suggestions.

I'll merge after you address.

**Comment by axiomcura** [comment]
@gwaybio addressed all suggestions!

**Review by gwaybio — APPROVED** [review]
👍

**Pull request merged**